edb/schema/objects.py

```diff
@@ -2018,7 +2018,26 @@ class InheritingObjectBase(Object):
     def _issubclass(
         self, schema: s_schema.Schema, parent: InheritingObjectBase
     ) -> bool:
-        lineage = compute_lineage(schema, self)
+        lineage_fast = [self]
+        lineage_fast.extend(self.get_ancestors(schema).objects(schema))
+        lineage_slow = compute_lineage(schema, self)
+        if lineage_fast != lineage_slow:
+            import pprint
+
+            pretty_fast = pprint.pformat(lineage_fast)
```

Comment:
We have a better `pprint`: 

```python
            import edb.common.debug
            edb.common.debug.dump(lineage_fast, lineage_slow, schema=schema)
```

when you pass `schema` it even resolves objects properly.